src/test/resources/README.md

```diff
@@ -0,0 +1,25 @@
+Incomplete test bags
+--------------------
+
+The sha's of the bags in the directory `vault` are most likely not valid.
+There might be other hic-ups with bag-store commands.
+The REST requests required for indexing should be OK.
+
+
+Copy the test bags on the local VM 
+----------------------------------
+
+Make sure to remove eventual files ignored by `.gitignore` such as `.DS_Store`
```

Comment:
@jo-pol the Dutch "eventuele" is in het Engels *niet* "eventual", maar "possible".

http://www.interglot.com/dictionary/nl/en/search?q=eventuele&m=key
http://www.interglot.com/dictionary/en/nl/search?q=eventual&m=key